site/en/guide/tpu.ipynb

```diff
@@ -250,18 +254,19 @@
         "id": "qYOYjYTg_31l"
       },
       "source": [
-        "### Input datasets\n",
-        "Efficient use of the `tf.data.Dataset` API is critical when using a Cloud TPU, as it is impossible to use the Cloud TPUs unless you can feed them data quickly enough. See [Input Pipeline Performance Guide](./data_performance.ipynb) for details on dataset performance.\n",
+        "### Load the dataset\n",
+        "\n",
+        "Efficient use of the `tf.data.Dataset` API is critical when using a Cloud TPU, as it is impossible to use the Cloud TPUs unless you can feed them data quickly enough. You can learn more about dataset performance in the [Input pipeline performance guide](./data_performance.ipynb).\n",
         "\n",
-        "For all but the simplest experimentation (using `tf.data.Dataset.from_tensor_slices` or other in-graph data) you will need to store all data files read by the Dataset in Google Cloud Storage (GCS) buckets.\n",
+        "For all but the simplest experimentations (using `tf.data.Dataset.from_tensor_slices` or other in-graph data) you will need to store all data files read by the Dataset in Google Cloud Storage (GCS) buckets.\n",
         "\n",
-        "For most use-cases, it is recommended to convert your data into `TFRecord` format and use a `tf.data.TFRecordDataset` to read it. See [TFRecord and tf.Example tutorial](../tutorials/load_data/tfrecord.ipynb) for details on how to do this. This, however, is not a hard requirement and you can use other dataset readers (`FixedLengthRecordDataset` or `TextLineDataset`) if you prefer.\n",
+        "For most use cases, you're recommended to convert your data into the `TFRecord` format and use a `tf.data.TFRecordDataset` to read it. Check the [TFRecord and tf.Example tutorial](../tutorials/load_data/tfrecord.ipynb) for details on how to do this. It is not a hard requirement and you can use other dataset readers, such as `FixedLengthRecordDataset` or `TextLineDataset`.\n",
```

Comment:
```suggestion
        "For most use cases, it is recommended to convert your data into the `TFRecord` format and use a `tf.data.TFRecordDataset` to read it. Check the [TFRecord and tf.Example tutorial](../tutorials/load_data/tfrecord.ipynb) for details on how to do this. It is not a hard requirement and you can use other dataset readers, such as `FixedLengthRecordDataset` or `TextLineDataset`.\n",
```

Please use full symbol names for `FixedLengthRecordDataset` and `TextLineDataset` to auto-link

Reply:
On it @lamberta 

⬇️

`tf.data.FixedLengthRecordDataset` <- `FixedLengthRecordDataset`
`tf.data.TextLineDataset` <- `TextLineDataset`